x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_disposed_creds_details.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiSpacer } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { BrowserFields } from '../../../../containers/source';
+import { Ecs } from '../../../../graphql/types';
+import { DraggableBadge } from '../../../draggables';
+
+import { PrimarySecondaryUserInfo } from './primary_secondary_user_info';
+import { SourceDest } from './source_dest_ip';
+
+const Details = styled.div`
+  margin-left: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+`;
+
+const TokensFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+export const AuditdDisposedLine = pure<{
+  id: string;
+  hostName: string | null | undefined;
+  userName: string | null | undefined;
+  primary: string | null | undefined;
+  secondary: string | null | undefined;
+  processExecutable: string | null | undefined;
+  processTitle: string | null | undefined;
+  workingDirectory: string | null | undefined;
+  args: string | null | undefined;
+  session: string | null | undefined;
```

Comment:
Thoughts on a separate type declaration for these?

Reply:
Sure thing.

I can add:

```ts
interface Props {
  id: string;
  hostName: string | null | undefined;
  userName: string | null | undefined;
  primary: string | null | undefined;
  secondary: string | null | undefined;
  processExecutable: string | null | undefined;
  processTitle: string | null | undefined;
  workingDirectory: string | null | undefined;
  args: string | null | undefined;
  session: string | null | undefined;
}

export const AuditdDisposedLine = pure<Props>(
```

And it should look better after that.